web/src/engine/osk/src/views/oskView.ts

```diff
@@ -521,11 +521,15 @@ export default abstract class OSKView
     // Set default OSK font size (Build 344, KMEW-90)
     // If the layer group specifies a fontsize value, we need
     // to apply that to the banner as well.
-    const layerFontSizeRaw = this.vkbd?.layerGroup?.spec.fontsize;
+    let layerFontSizeRaw = this.vkbd?.layerGroup?.spec.fontsize;
+    if(layerFontSizeRaw == '') {
```

Comment:
```suggestion
    if(layerFontSizeRaw === '') {
```

Just a little more explicit?

Reply:
I'm not actually sure we want this, but just wondering if any type coercion here could lead to unexpected equality and different outcomes?